dohq_artifactory/admin.py

```diff
@@ -175,7 +196,8 @@ def _read_response(self, response):
         """
         # self.password = ''  # never returned
         self.name = response['name']
-        self.email = response['email']
+        if "email" in response.keys():
```

Comment:
May be like this? 
```python
self.email = response.get('email', None)
```

Reply:
`self.email = response.get('email')` is sufficient